samples/bluetooth/broadcast_audio_sink/overlay-bt_ll_sw_split.conf

```diff
@@ -0,0 +1,9 @@
+# Enable support for Broadcast ISO Sync
+CONFIG_BT_CTLR_SYNC_ISO=y
+
+# Supports the highest SDU size required by any BAP LC3 presets (155)
+CONFIG_BT_CTLR_SYNC_ISO_PDU_LEN_MAX=155
+
+# Supports the highest advertising data that is set in a single HCI command in
+# Zephyr Bluetooth Controller
+CONFIG_BT_CTLR_SCAN_DATA_LEN_MAX=191
```

Comment:
The idea of moving from board configs to overlays, makes it easier to define a single overlay for multiple boards, but it also makes it less intuitive to use. Board configs are automatically applied when building for a board, and the user does not need to do anything besides just doing a `west build` for the board. However, with the overlay, the user needs to know how to and when to add the overlay to the build. 

I'm not sure this is the right direction to go with these config files.

Reply:
Very few (direction finding, audio, and controller-only) upstream samples actually use board specific config files. As there is no difference in the duplicated configs for multiple boards, but rather it is about building host+controller combined binary, overlay conf file to include Controller (in-tree or out-of-tree) seems right to me.

Reply:
That's a fair point :)